src/test/scala/io/phdata/retirementage/filters/CustomTableFilterTest.scala

```diff
@@ -0,0 +1,55 @@
+package io.phdata.retirementage.filters
+
+import io.phdata.retirementage.domain._
+import io.phdata.retirementage.storage.HdfsStorage
+import io.phdata.retirementage.{SparkTestBase, TestObjects}
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.types._
+import org.scalatest.FunSuite
+import java.sql.Date
+
+class CustomTableFilterTest extends FunSuite with SparkTestBase {
+  test("Filter out a date when column is unix time seconds") {
+
+    val table    = CustomTable("table1", "parquet", List(CustomFilter(s"date = ${Date.valueOf("2018-12-25").getTime / 1000}")), None, None)
```

Comment:
Maybe pull the filtered value out into a variable? Otherwise it's hard to see what's being tested.